src/lib/utils/events.ts

```diff
@@ -309,22 +314,22 @@ export class EventDispatcher {
     /**
      * Map of all handlers registered with the "on" function.
      */
-    private _events: EventHandlers;
+    private _events?: EventHandlers;
 
     /**
      * Map of all objects this instance is listening to.
      */
-    private _listeningTo: EventListeners;
+    private _listeningTo?: EventListeners;
 
     /**
      * Map of all objects that are listening to this instance.
      */
-    private _listeners: EventListeners;
+    private _listeners?: EventListeners;
 
     /**
      * A unique id that identifies this instance.
      */
-    private _listenId: string;
+    private _listenId!: string;
```

Comment:
I think this can be `_listenId?`, as the uses on lines 408 and 415 check for null?